docs/running-node.md

```diff
@@ -107,6 +107,17 @@ curl -L -O https://raw.githubusercontent.com/zigbee-alliance/distributed-complia
 </details>
 <!-- markdownlint-enable MD033 -->
 
+> Note:
```

Comment:
I think we need a similar Note in the switch_to_cosmovisor how-to docs.

Reply:
Added